cli/src/main/java/com/devonfw/tools/ide/tool/mvn/Mvn.java

```diff
@@ -41,6 +42,12 @@ public class Mvn extends PluginBasedCommandlet {
   /** The name of the settings-security.xml */
   public static final String SETTINGS_SECURITY_FILE = "settings-security.xml";
 
+  /** The mvn wrapper file name. */
+  public static final String MVN_WRAPPER_FILENAME = "mvnw";
```

Comment:
Just for the record: There is also `mvnw.cmd` created by https://maven.apache.org/wrapper/maven-wrapper-plugin/usage.html[maven-wrapper-plugin].
IMHO we do not need to handle that specifically.
I would consider it an odd edge-case if `mvnw.cmd` is present but `mvnw` bash script is not.
As we also always require `bash` as pre-requisite and already have logic to detect and execute bash scripts, there is nothing else to do here. I just wanted to point this out. Will close the comment to prevent blocking the merge.